jsonrpc/eth_endpoint.go

```diff
@@ -388,19 +390,9 @@ func (e *Eth) GetStorageAt(
 	index types.Hash,
 	filter BlockNumberOrHash,
 ) (interface{}, error) {
-	var (
-		header *types.Header
-		err    error
-	)
-
-	// The filter is empty, use the latest block by default
-	if filter.BlockNumber == nil && filter.BlockHash == nil {
-		filter.BlockNumber, _ = createBlockNumberPointer("latest")
-	}
-
-	header, err = e.getHeaderFromBlockNumberOrHash(&filter)
+	header, err := e.getHeaderFromBlockNumberOrHash(filter)
 	if err != nil {
-		return nil, fmt.Errorf("failed to get header from block hash or block number")
+		return nil, err
```

Comment:
Could you add error wrapping there with block number and hash?

Reply:
It is already done as part of `getHeaderFromBlockNumberOrHash`. No need to do it every time now.

Reply:
https://github.com/0xPolygon/polygon-edge/pull/875/files#diff-28b959cf43f3cb0ae8b7ac87892a8dbb0e6792100e04beb3d20b14dd18c88487R104